<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitByRuleWidget</class>
 <widget class="QWidget" name="SplitByRuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="columnComboBox">
     <item>
      <property name="text">
       <string>User</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Task</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Description</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="operatorComboBox">
     <item>
      <property name="text">
       <string>==</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="comparisonValueEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="--error" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
